### ISSUE TITLE
ReadOnly

### ISSUE THREAD

**Issue by SharpNoiZy** [Ericsson/exchangecalendar#229]
Hi,

Did I do something wrong or is it correct that your add on just adds Exchange calendar in ReadOnly mode?

Because Thunderbird/Lighting told me so, and if I untick the box in the preferences, close the window and open it again the box is checked again.

Kind regards

**Comment by pakezonite** [comment]
I would also like to know this. Before updating to Lightning version 4.0.0.1 and EWS 3.4.0 beta3 I could still read and write -know turned in to read only and won't accept changes.

**Comment by bavincen** [comment]
please check latest version https://github.com/Ericsson/exchangecalendar/wiki/How-to-Install-latest-addon

**Comment by pakezonite** [comment]
Just build and installed the latest version and still read-only!

Also tried re-creating (re-subscibing to) the calendar, but I can't get write access.

**Comment by bavincen** [comment]
please check with admin.. this is not seems to be addon issue.. or try with new profile

**Comment by pakezonite** [comment]
Hi!

Just thought I would let you know (or others with similar issues) that I solved it:

Before upgrading to Lightning version 4.0.0.1 and EWS 3.4.0 beta3 in the settings If I put the email address as the username (including the domain in it) the settings wouldn't work and I had to set the username to the SamAccountName (different from the email address) and give the domain separately
(the domain name I had to give was not the same as the end part of my email address).

After upgrading to Lightning version 4.0.0.1 and EWS 3.4.0 beta3 those settings working before only gave read-only access and I had to switch to using the email address as the username.

So, maybe something changed in the way the login is handled in the code because both versions new and old still work and the service provider didn't change anything on the server side.

Hope this is helpful to someone... 

Best regards
